src/handler/mod.rs

```diff
@@ -0,0 +1,106 @@
+use anyhow::{Error, Result};
+use figlet_rs::FIGfont;
```

Comment:
I thought we decided not to rely on `figlet` for the MOTD?

Reply:
+1